daemon/pool.go

```diff
@@ -24,9 +24,8 @@ var (
 	// scaling policy (see DefaultScalingPolicy()).
 	DefaultMaxInstancesPerDriver = runtime.NumCPU()
 
-	ErrPoolClosed         = errors.NewKind("driver pool already closed")
-	ErrInvalidPoolTimeout = errors.NewKind("invalid timeout: %v")
-	ErrNegativeInstances  = errors.NewKind("cannot set instances to negative number")
+	ErrPoolClosed        = errors.NewKind("driver pool already closed")
```

Comment:
Please document these exported names!

Reply:
Will do in the next PR.

Reply:
It would make sense to include at least a doc for the new error in here

Reply:
Did not realize https://github.com/bblfsh/bblfshd/pull/258/files#diff-ef2ae0497174a079d805f1c50a0f2444 is already a thing; 👍